xmtp_mls/src/verified_key_package_v2.rs

```diff
@@ -64,6 +80,21 @@ impl VerifiedKeyPackageV2 {
     pub fn hpke_init_key(&self) -> Vec<u8> {
         self.inner.hpke_init_key().as_slice().to_vec()
     }
+
+    pub fn life_time(&self) -> Option<VerifiedLifetime> {
+        let lifetime_result = panic::catch_unwind(AssertUnwindSafe(|| {
+            self.inner.life_time() // This might panic
```

Comment:
given our uncertainty over whether lifetimes were set, and the existence of [this issue,](https://github.com/openmls/openmls/issues/1663) I added this panic catch to be safe. This way if we make lifetimes optional at some point in the future this would not panic in older versions. happy to remove if this seems excessive.

Reply:
great catch